modules/k8s-nmstate-troubleshooting-dns-disconnected-env.adoc

```diff
@@ -0,0 +1,118 @@
+// Module included in the following assemblies:
+//
+// * networking/k8s_nmstate/k8s-nmstate-troubleshooting-node-network.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="troubleshooting-dns-disconnected-env_{context}"]
+= Troubleshooting DNS connectivity issues in a disconnected environment
+
+If you experience DNS connectivity issues when configuring `nmstate` in a disconnected environment, you can configure the DNS server to resolve the list of name servers for the domain `root-servers.net`.
+
+== Configuring the bind9 DNS named server  
+
+For a cluster configured to query a `bind9` DNS server, you can add the `root-servers.net` zone to a configuration file that contains at least one NS record.
+For example you can use the `/var/named/named.localhost` as a zone file that already matches this criteria.
+
+.Procedure
+
+. Add the `root-servers.net` zone at the end of the `/etc/named.conf` configuration file by running the following command:
++
+[source,terminal]
+----
+$ cat >> /etc/named.conf <<EOF
+zone "root-servers.net" IN {
+    	type master;
+    	file "named.localhost";
+};
+EOF
+----
+
+. Restart the `named` service by running the following command:
++
+[source,terminal]
+----
+$ systemctl restart named
+----
+
+. Confirm that the `root-servers.net` zone is present by running the following command:
++
+[source,terminal]
+----
+$ journalctl -u named|grep root-servers.net
+----
++
+.Example output
++
```

Comment:
I don't think you need the carriage return and `+` after the `.Example output` heading. I don't see a hard rule; but that seems to be the way [it is presented in the doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#source-tags-for-terminal-commands-and-output).

Reply:
thanks for the review @mburke5678 
I removed that extra line and `+`